util/chunk/codec.go

```diff
@@ -163,25 +162,6 @@ func (c *Codec) bytesToI32Slice(b []byte) (i32s []int32) {
 	return i32s
 }
 
-// varElemLen indicates this column is a variable length column.
-const varElemLen = -1
-
-func getFixedLen(colType *types.FieldType) int {
```

Comment:
This is the length in Chunk, but the length in the storage is different.

Reply:
Yes, but we do not maintain the length for fixed length column.

Reply:
I mean we should use a different function to estimate the average column size in the storage.